InfiniteRecharge/src/main/java/frc/robot/RobotMap.java

```diff
@@ -21,7 +27,11 @@
  * - none of their code must change if there is a change in devices used
  * - we can run tests on each subsystem's logic using simulated devices
  */
+
 public class RobotMap {
+    public static final int kIntakeCanID = 9;
+    public static final int kItntakeActuatorCanID = 10;
```

Comment:
are we doing all port & can id assignments in here? we should move all of them to one spot rather than some in Constants and some in here

Reply:
* for a later pr

Reply:
I have put all the can ids in the constants for now, we will have to think about where we want them in the long term. I also added a PbVictor888 for all our victor 888 needs.